### PR TITLE
first commit cosmos/code/language/c/loop/break.c

### PR DESCRIPTION
**Pull request by raksha009**
**Fixes issue:**

First Commit

**Changes:**

Added comments/ documentation for the break statement code in c language

### CODE REVIEW

**Comment by raksha009** [comment]
@arnavb plz review my PR

**Comment by raksha009** [comment]
@arnavb  plz review this PR

File: code/language/c/loop/break.c [hunk 1]
```diff
@@ -1,3 +1,7 @@
+// Defining a pre-processor directive
```
Comment (arnavb) [inline code/language/c/loop/break.c]: These four are not needed.

File: code/language/c/loop/break.c [hunk 2]
```diff
@@ -7,6 +11,10 @@ int main()
         printf("value of variable num is: %d\n", num);
         if (num==2)
         {
+		/* 
+		Break statement stops the execution of the program when num = 2.
```
Comment (arnavb) [inline code/language/c/loop/break.c]:
```suggestion
		Break statement stops the execution of the loop when num = 2. 
```

**Comment by raksha009** [comment]
@arnavb done the changes.

**Comment by raksha009** [comment]
Thanks sir

**Pull request merged**